queries.yaml

```diff
@@ -231,12 +237,11 @@ post_process:
       property_keys: [name, kind]
       geometry_types: [Point]
       min_distance: 256.0
-  - fn: TileStache.Goodies.VecTiles.transform.drop_features_where
+  - fn: TileStache.Goodies.VecTiles.transform.drop_properties
```

Comment:
Nice catch! I thought the intent actually was to only drop the property. I'll go ahead and correct this.